src/tls.c

```diff
@@ -10203,6 +10524,38 @@ static int TLSX_PopulateSupportedGroups(WOLFSSL* ssl, TLSX** extensions)
         #endif
 #endif
 
+#ifdef HAVE_LIBOQS
+    ret = TLSX_UseSupportedCurve(extensions, WOLFSSL_KYBER512, ssl->heap);
+    if (ret != WOLFSSL_SUCCESS) return ret;
```

Comment:
It would be cleaner to rewrite this so that the return at the end on the function is used. This will follow the way that we do it in other parts of wolfSSL and reduce the amount of return points.
```
    ret = TLSX_UseSupportedCurve(extensions, WOLFSSL_KYBER512, ssl->heap);
    if (ret == WOLFSSL_SUCCESS)
        ret = TLSX_UseSupportedCurve(extensions, WOLFSSL_KYBER768, ssl->heap);
    if (ret == WOLFSSL_SUCCESS)
        ret = TLSX_UseSupportedCurve(extensions, WOLFSSL_KYBER1024, ssl->heap);
    if (ret == WOLFSSL_SUCCESS)
        ret = TLSX_UseSupportedCurve(extensions, WOLFSSL_NTRU_HPS2048509,
                                     ssl->heap);
    if (ret == WOLFSSL_SUCCESS)
        ret = TLSX_UseSupportedCurve(extensions, WOLFSSL_NTRU_HPS2048677,
                                     ssl->heap);
...
```

Reply:
Yup.  I can do that.

Reply:
So I changed it, but I have to say I think its a bit confusing to me.